unison-cli/package.yaml

```diff
@@ -44,7 +44,7 @@ dependencies:
   - ki
   - lens
   - lock-file
-  - lsp >= 2.2.0.0
```

Comment:
`lsp` has a couple of small breaking changes that we depend on:
* https://github.com/haskell/lsp/pull/552
* https://github.com/haskell/lsp/pull/523